OpenEdXMobile/src/main/java/org/edx/mobile/util/Config.java

```diff
@@ -513,6 +514,22 @@ public boolean areNotificationsEnabled() {
         }
     }
 
+    public static class EmbeddedYoutubeConfig {
+        @SerializedName("ENABLED")
+        private boolean mEnabled;
```

Comment:
This class field should not start with `m` as app convention.

Reply:
@farhan-arshad-dev  following this  [documentation](https://source.android.com/setup/contribute/code-style#follow-field-naming-conventions), the private variables should start with m and the others classes in this file start with m, is there any update or special reason to modify this ??

Reply:
@andrey-canon This is a convention that we don't like to follow in this project cuz the IDEs that we use these days are modern enough to differentiate between different kinds of variables and their types.

Having said that, there certainly are usages of this convention within the code but that code was written a while back or we haven't gotten the chance to refactor it just yet.

Reply:
Got it